Framework/Crystal/test/PredictPeaksTest.h

```diff
@@ -216,4 +247,54 @@ class PredictPeaksTest : public CxxTest::TestSuite {
   }
 };
 
+class PredictPeaksTestPerformance : public CxxTest::TestSuite {
+public:
+  void test_many_peaks_rectangular() {
```

Comment:
Naming is a typo yes. 

It's a performance test, so we're checking for speed and not correctness. I could add a basic assertion but it shouldn't have detailed checking like a unit test.